phys2bids/physio_obj.py

```diff
@@ -325,6 +337,79 @@ def print_info(self, filename):
 
         LGR.info(info)
 
+    def check_trigger_amount_std(self, chtrig=1, num_timepoints_expected=0, tr=0):
```

Comment:
Should this function be deleted now, since you combined its functionality into `check_trigger_amount()`?

Reply:
yeah i left it while testing in case i needed it